apps/console/src/features/applications/components/help-panel/overview.tsx

```diff
@@ -106,10 +106,13 @@ export const HelpPanelOverview: FunctionComponent<HelpPanelOverviewPropsInterfac
     //     if (samlConfigurations !== undefined) {
```

Comment:
Can't we get rid of this commented block.

Reply:
Yes, we can.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1968/commits/a829b59a7fd9df20ef6785dce95839bde3dbcd5e.